cloudimg/aws.py

```diff
@@ -632,6 +658,25 @@ def import_snapshot(self, obj, snapshot_name, tags=None):
 
         return snapshot
 
+    def copy_ami(self, image_id, image_name, image_region, tags=None):
```

Comment:
What's the `tags` argument about?  It's not documented and not used.

Reply:
Did it for testing, missed to remove it. Deleting now.